### PR TITLE
Issues/kp 353 new search third cycle courses

### PR DESCRIPTION
**Pull request by amirhossein-haerian**
The address for third-cycle courses search is http://localhost:3000/utbildning/forskarutbildning/kurser/sok-ny-design?l=en

I asked Joel about the search footers, inside the third-cycle courses search, to know if we need to keep it or not. (I will do the required changes later when I know that I should keep it or not)

### CODE REVIEW

**Comment by amirhossein-haerian** [comment]
The search footer is now added for both (all courses search and third-cycle courses search)

File: public/js/app/pages/NewSearchLandingPage.tsx [hunk 1]
```diff
@@ -57,30 +55,87 @@ const NewSearchLandingPage = () => {
...
-      </MainContent>
+        <CollapseDetails title={collapseHeader}>
+          <HelpTexts {...helptextsProps} />
+        </CollapseDetails>
+        {searchMode === SEARCH_MODES.thirdCycleCourses && (
+          <Row>
+            <Link href={courseSearchLink('sokkurs', language)}>{linkToUsualSearch}</Link>
+          </Row>
+        )}
+        <Row>
```
Comment (karlandindrakryggen) [inline public/js/app/pages/NewSearchLandingPage.tsx:135]: I believe that sometimes we are using the `<Row>` component where it is unnecessary, like in this case (and the lines above) it doesn't really add anything?

**Review by karlandindrakryggen — APPROVED** [review]
LGTM! ⭐️ 

Just one small thought, but nothing you need to change if you don't want to :)

File: public/js/app/pages/NewSearchLandingPage.tsx (hunk 1 above)
Comment (amirhossein-haerian) [reply public/js/app/pages/NewSearchLandingPage.tsx:135]: I agree with you, it just gives us "flex" display.
I have tested without having Row and made some tiny changes in scss; it works well so I removed them.

**Pull request merged**